src/Util/Caller.js

```diff
@@ -1,17 +1,27 @@
+import Cookies from 'universal-cookie';
 
+const cookies = new Cookies();
 
-const url_backend ="https://park-inn-ispp-be.herokuapp.com/"
-const url_frontend = "https://park-inn-ispp-fe.herokuapp.com/"
+const url_backend ="https://park-inn-ispp-be.herokuapp.com"
+const url_frontend = "https://park-inn-ispp-fe.herokuapp.com"
 
 
      async function call(path_to_call,method,body){
+        let headers_ = {'Content-Type': 'application/json', 'Access-Control-Allow-Origin' : url_frontend, "mode": "cors"}
+        const token = await cookies.get('AuthToken')
+
+        if(token!=undefined && token!="undefined"){
+            headers_ = {'Authorization' : 'Bearer ' + token.toString(),'Content-Type': 'application/json', 'Access-Control-Allow-Origin' : url_frontend, "mode": "cors"}
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9156896)
```suggestion
            headers_ = {"Authorization" : 'Bearer ' + token.toString(),'Content-Type': 'application/json', 'Access-Control-Allow-Origin' : url_frontend, "mode": "cors"}
```